docs/UserGuide.md

```diff
@@ -448,6 +448,7 @@ companies less clustered and thus more organised.
 * Refer to [Demo Use Case #2](#usecase2) for an example use case.
 
 **Examples:**
+>>>>>>> 183b65a12c384a90ad9bcfc0ea458d5e32a94af1
```

Comment:
I think you forget to remove this line after conflict.

Reply:
Good catch thanks!